package.json

```diff
@@ -2,18 +2,15 @@
   "name": "stimulus_reflex_table_filter",
   "private": true,
   "dependencies": {
-    "@rails/actioncable": "^6.0.0-alpha",
-    "@rails/activestorage": "^6.0.0-alpha",
-    "@rails/ujs": "^6.0.0-alpha",
+    "@rails/actioncable": "^6.0.0",
     "@rails/webpacker": "^4.0.7",
     "dom-confetti": "^0.1.1",
     "stimulus": "^1.1.1",
-    "stimulus_reflex": "^2.1.2",
-    "turbolinks": "^5.2.0"
```

Comment:
Same comment here, what’s the value of removing these? I realize they aren’t being used but like I said in another comment I want this to be as close to the rails defaults as possible. Mostly bc I don’t want to have to call out/explain the changes in a blog post.